File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/CoreWS/CoreWS_Generic.ts
import { CoreWS_GenericObjP } from './CoreWS_GenericObjP';
import { CoreWS_GenericPayload } from './CoreWS_GenericPayload';

export class CoreWS_Generic<T> extends CoreWS_GenericPayload{

    InData: T;

    constructor(objP: CoreWS_GenericObjP, InData: T) {
        super(objP);
        this.InData = InData;
    }
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreDTOStd/CoreWS_Generic.cs
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.CompilerServices;
using System.Security;
using System.Text;
using System.Threading.Tasks;
using Microsoft.VisualBasic;

public class CoreWS_Generic<T> : CoreWS_GenericPayload
{
    public T InData { get; set; }

    public CoreWS_Generic(CoreWS_GenericObjP objP, T inData) : base(objP)
    {
        this.InData = inData;
    }

    public CoreWS_Generic() { }

}